tests/science/science.cfg

```diff
@@ -1 +1,21 @@
-[test]
+[science_test_snotel]
+driver = classic
+test_description = snotel point simulations
+global_parameter_file = global_param.classic.5.0_snotel.txt
+expected_retval = 0
+check = not_implemented
+plots = water_year
+observations_path = snotel
+vic4.2.d = vic_4_2
+vic5.0.0 = vic_5_0
```

Comment:
I think that's a great idea. I'll refactor it so that we can easily add previous versions or even branches.